src/components/Epg/Epg.tsx

```diff
@@ -0,0 +1,90 @@
+import { Epg as EpgContainer, Layout } from 'planby';
+import React, { useMemo } from 'react';
+import { useTranslation } from 'react-i18next';
+import { isBefore, subHours } from 'date-fns';
+
+import styles from './Epg.module.scss';
+
+import type { Config } from '#types/Config';
+import Button from '#src/components/Button/Button';
+import EpgChannelItem from '#src/components/EpgChannel/EpgChannelItem';
+import EpgProgramItem from '#src/components/EpgProgramItem/EpgProgramItem';
+import EpgTimeline from '#src/components/EpgTimeline/EpgTimeline';
+import Spinner from '#src/components/Spinner/Spinner';
+import type { EpgChannel, EpgProgram } from '#src/services/epg.service';
+import ChevronRight from '#src/icons/ChevronRight';
+import ChevronLeft from '#src/icons/ChevronLeft';
+import usePlanByEpg from '#src/hooks/usePlanByEpg';
+import useBreakpoint, { Breakpoint } from '#src/hooks/useBreakpoint';
+
+type Props = {
+  channels: EpgChannel[];
+  setActiveChannel: (id: string, programId?: string | undefined) => void;
+  channel: EpgChannel | undefined;
+  program: EpgProgram | undefined;
+  config: Config;
+};
+
+export default function Epg({ channels, setActiveChannel, channel, program, config }: Props) {
+  const breakpoint = useBreakpoint();
+  const { t } = useTranslation('common');
+
+  const isSmall = breakpoint < Breakpoint.sm;
```

Comment:
```suggestion
  const isSmall = breakpoint <= Breakpoint.sm;
```
Otherwise, rename the const to `isExtraSmall`.  We've now used this same pattern in several places, would it make sense to add it to the hook?

Reply:
Yes, I agree. The could look something like:

```ts
const { breakpoint, isMobile, isTablet, isDesktop } = useBreakpoint();

if (isTablet) {

// or
const { breakpoint, size } = useBreakpoint();

if (size === 'tablet')
```

Reply:
Renamed to `isMobile` for now.